src/routes/clients.test.js

```diff
@@ -0,0 +1,68 @@
+const {
+  constants: { CLIENT_KINDS },
+} = require('@notify-watcher/core');
+const request = require('../tests/supertest');
+const User = require('../models/user');
+const { HTTP_CODES } = require('../constants');
+
+describe('clients routes', () => {
+  describe('POST clients/register', () => {
+    const register = body => request.post('/clients/register').send(body);
+
+    describe('for a non existent user', () => {
+      it('should return "not found"', () =>
+        register({ email: 'invalid email' }).expect(HTTP_CODES.notFound));
+    });
+
+    describe('for an existent user', () => {
+      const clientData = { kind: CLIENT_KINDS.telegram };
+      let user;
+      let token;
+
+      beforeAll(async () => {
+        user = await User.create({ email: 'user-client-register@example.org' });
+        await user.createSecret();
+        token = await user.generateToken();
+      });
+
+      afterAll(() => user.deleteOne());
```

Comment:
we should use `deleteOne` with Model, not instances. So we don't have concussion between `User.deleteOne` and `user.deleteOne`

```suggestion
      afterAll(() => user.remove());
```